config/routes.rb

```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  # Guest
+  get "guest/:id", to: "guests#show"
+  post "guest/:id", to: "guest_rooms#create"
```

Comment:
We want our routs to be named similarly to their resources (plural). So, something like '/guests/:id'